js/views/userPage/Store.js

```diff
@@ -63,6 +64,7 @@ export default class extends BaseVw {
 
     this.listenTo(app.settings, 'change:country', () => (this.showShippingChangedMessage()));
     this.listenTo(app.settings, 'change:localCurrency', () => (this.showDataChangedMessage()));
+    this.listenTo(app.profile, 'change:vendor', () => (this.showDataChangedMessage()));
```

Comment:
Can we just toggle a 'hide' class on that 'your store is currently'... box? Seems like a lot smoother update than showing this data changed pop-up and even if, on remote servers, the change happens with a lag and the user is doing something on the page... doesn't seem like it would be so distracting.